[2024-05-02T10:14:40.218+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-02T10:14:40.245+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3-data-transfer.transfer-customer scheduled__2024-05-01T00:00:00+00:00 [queued]>
[2024-05-02T10:14:40.255+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3-data-transfer.transfer-customer scheduled__2024-05-01T00:00:00+00:00 [queued]>
[2024-05-02T10:14:40.256+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-02T10:14:40.268+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): transfer-customer> on 2024-05-01 00:00:00+00:00
[2024-05-02T10:14:40.277+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2142) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-02T10:14:40.278+0000] {standard_task_runner.py:63} INFO - Started process 2156 to run task
[2024-05-02T10:14:40.279+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 's3-data-transfer', 'transfer-customer', 'scheduled__2024-05-01T00:00:00+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/new-dag.py', '--cfg-path', '/tmp/tmp8b3jtcvj']
[2024-05-02T10:14:40.280+0000] {standard_task_runner.py:91} INFO - Job 83: Subtask transfer-customer
[2024-05-02T10:14:40.323+0000] {task_command.py:426} INFO - Running <TaskInstance: s3-data-transfer.transfer-customer scheduled__2024-05-01T00:00:00+00:00 [running]> on host 0a71bd2d4f31
[2024-05-02T10:14:40.402+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3-data-transfer' AIRFLOW_CTX_TASK_ID='transfer-customer' AIRFLOW_CTX_EXECUTION_DATE='2024-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-01T00:00:00+00:00'
[2024-05-02T10:14:40.403+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-02T10:14:41.172+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-02T10:14:41.173+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-02T10:14:41.181+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=s3-data-transfer, task_id=transfer-customer, execution_date=20240501T000000, start_date=20240502T101440, end_date=20240502T101441
[2024-05-02T10:14:41.215+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-02T10:14:41.232+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-02T10:14:41.234+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
